### ISSUE TITLE
Issue requiring workaround when including package into an executable built with pyinstaller

### ISSUE THREAD

**Issue by porteri** [iamlikeme/rainflow#65]
No package metadata is found for rainflow.

```
Traceback (most recent call last):
  File "importlib\metadata\__init__.py", line 563, in from_name
StopIteration

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "exe_interface.py", line 24, in <module>
  File "rainflow.py", line 15, in <module>
  File "importlib\metadata\__init__.py", line 1008, in version
  File "importlib\metadata\__init__.py", line 981, in distribution
  File "importlib\metadata\__init__.py", line 565, in from_name
importlib.metadata.PackageNotFoundError: No package metadata was found for rainflow
```

**Comment by iamlikeme** [comment]
I don't know pyinstaller in and out, so you need to provide some more context. E.g. what command you are running, which Python version, etc.

**Comment by porteri** [comment]
python 3.11.2
pyinstaller 5.8.0

To build:
```
pyinstaller ./setup.spec
```

The error above occurs when the built executable (no error during build steps) is loaded.

There is a workaround for anyone who uses `pyinstaller` with a `setup.spec` file. A clip of what works in my case is shown below.

```
from PyInstaller.utils.hooks import copy_metadata

# Workaround for rainflow 3.1.1
rainflow_data = copy_metadata('rainflow')
# End of workaround

a = Analysis(
     ...
     datas = [my_data_files] + rainflow_data
     ...)
```

**Comment by iamlikeme** [comment]
Solved in #66, and released in 3.2.0
